setup.cfg

```diff
@@ -33,7 +33,7 @@ setup-requires =
 install-requires =  # ?? requires-dist?
     pbr
     numpy
-    networkx ~= 2.0
+    networkx ~= 3.0
```

Comment:
seeing that there is only one test, which was changed, would it be an option to also allow networkx v2 and raise a deprecation warning for some time until we cement the change? I'm a bit worried about the projects using martinize2 such as the MAD and Go script etc.

Reply:
How about `>= 2.0` and no further warnings or checks?

Reply:
sounds good